ColqueOlimpia/Outfitaplication/src/GeneralClasses/Closet.java

```diff
@@ -0,0 +1,8 @@
+package GeneralClasses;
+
+import java.util.Collection;
+
+public class Closet {
```

Comment:
besides the clothesCollection you will need an outfit collection.
You have to add all the necessary methods to manipulate this two collections, addClothes, addOutfit, etc.